fineract-provider/src/main/java/org/apache/fineract/infrastructure/creditbureau/domain/CreditBureau.java

```diff
@@ -31,29 +30,30 @@
 @Table(name = "m_creditbureau")
 public class CreditBureau extends AbstractPersistableCustom {
 
-
+    @Column(name = "name", nullable = false)
```

Comment:
I'm curious and would like to learn why this is required? the name is the name, no change. The nullable false why? Because it's NOT NULL in the DB schema? Does EclipseLink actually validate that? This, and some of the other changes you seem to have to make (below), IMHO would be a clean up change worth making in a smaller separate PR already...